capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -38,34 +38,46 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.sql.DataSource;
 
-@WebServlet("/study_set")
+@WebServlet(urlPatterns = { "/study_set/*" })
 public class StudySetServlet extends HttpServlet {
-  private final String search_sql_statement = 
+  private final String SEARCH_SQL_STATEMENT =
```

Comment:
Good call changing this to uppercase: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names